include/vkfw/vkfw.hpp

```diff
@@ -3130,8 +3130,11 @@ namespace VKFW_NAMESPACE {
     glfwCreateWindowSurface(instance, window, allocator, &output);
 
     vk::Optional<vk::AllocationCallbacks const> vulkan_hpp_allocator = nullptr;
-    if (allocator)
-      vulkan_hpp_allocator = vk::AllocationCallbacks(*allocator);
+    if (allocator) {
+      vk::AllocationCallbacks callbacks;
+      callbacks = *allocator;
+      vulkan_hpp_allocator = callbacks;

```

Comment:
Ah, wait, I think there's a cleaner way to represent this!

```cpp
vulkan_hpp_allocator = vk::AllocationCallbacks{};
*vulkan_hpp_allocator = *allocator;
```

What do you think?

Reply:
Ah, right, `const`! Sorry for the noise then, let's go with `tmp`.

Reply:
BTW, it probably doesn't work because `*vulkan_hpp_allocator` returns `const vk::AllocationCallbacks`, which is not assignable (by the virtue of being `const`).